server/app/mqtt.py

```diff
@@ -63,7 +63,7 @@ async def helper(sensor_identifier, revision, configuration):
             try:
                 # Try to publish the configuration
                 await mqttc.publish(
-                    topic=f"configurations/{sensor_identifier}",
+                    topic=f"{settings.MQTT_BASE_TOPIC}configurations/{sensor_identifier}",
```

Comment:
Could there be a / missing? Does it make sense to join these with os.path?

Reply:
the env var `MQTT_BASE_TOPIC` should include a trailing slash.
It can however also be left empty, in which case `{settings.MQTT_BASE_TOPIC}configurations/{sensor_identifier}` still works, while `{settings.MQTT_BASE_TOPIC}/configurations/{sensor_identifier}` would fail

Reply:
I can make this more verbose, too. But by default this env-var isn't set anyways